fontir/src/ir.rs

```diff
@@ -746,18 +747,11 @@ pub struct NameBuilder {
     version_minor: u32,
 }
 
-impl Default for NameBuilder {
-    fn default() -> Self {
-        let mut builder = Self {
-            names: Default::default(),
-            name_to_key: Default::default(),
-            version_major: Default::default(),
-            version_minor: Default::default(),
-        };
-        // Based on diffing fontmake Oswald build
-        builder.add(NameId::SUBFAMILY_NAME, "Regular".to_string());
-        builder
-    }
+const STYLE_MAP_STYLE_NAMES: [&str; 4] = ["regular", "bold", "italic", "bold italic"];
+
+/// Returns true if the style name is one of the four standard style names.
+fn is_ribbi(style_name: &str) -> bool {
+    return STYLE_MAP_STYLE_NAMES.contains(&style_name.to_lowercase().as_str());
```

Comment:
but if _this_ is the only place where you're referencing this static than I would actually just keep it inline in this function, either by having the `static` declared here or by just writing,

```rust
["regular", "etc"].contains(&thing)
```

and just for fun you could also express this using `match`, which _might_ generate slightly better code (very unlikely to matter in this case):

```rust
matches!(
        style_name.to_lowercase().as_str(),
        "regular" | "bold" | "italic" | "bold italic"
    )
```

Reply:
oh good call, +1 to `matches!` if that works

Reply:
I went for `matches!`